packages/react-devtools/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "react-devtools",
+  "productName": "React Developer Tools",
   "version": "2.0.12",
   "description": "Use react-devtools outside of the browser",
+  "main": "./packages/react-devtools/app.js",
```

Comment:
Seems like `electron-packager` gets upset when it can't find those window definitions, which is why I pointed it to the `app.js`:

<img width="532" alt="screen shot 2017-02-22 at 3 10 53 pm" src="https://cloud.githubusercontent.com/assets/1720093/23230237/366015e8-f911-11e6-81e9-a7b6bbb113e5.png">

That bin script just requires it, though, no idea why `electron-packager` is barfing on it ☹️